pkg/controllers/clusterinventory/clusterprofile/controller.go

```diff
@@ -254,9 +256,10 @@ func (r *Reconciler) cleanupClusterProfile(ctx context.Context, clusterName stri
 // SetupWithManager sets up the controller with the controller manager.
 func (r *Reconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
+		WithOptions(ctrlruntime.Options{SkipNameValidation: ptr.To(true)}).
```

Comment:
better to give this controller a unique name instead of trying to work around that